dbms/src/Debug/dbgFuncCoprocessor.cpp

```diff
@@ -183,155 +183,166 @@ void setTipbRegionInfo(coprocessor::RegionInfo * tipb_region_info, const std::pa
     range->set_end(RecordKVFormat::genRawKey(table_id, handle_range.second.handle_id));
 }
 
-BlockInputStreamPtr executeQuery(Context & context, RegionID region_id, const DAGProperties & properties, QueryTasks & query_tasks, MakeResOutputStream & func_wrap_output_stream)
```

Comment:
Just split exeucteQuery into executeMPPQuery and executeNonMPPQuery

Reply:
What is the difference between the two here

Reply:
Before split, I should pass func_wrap_output_stream to call executeQuery since executeNonMPPQuery uses it